clients.asciidoc

```diff
@@ -155,6 +155,7 @@ For other operating systems, use the package manager of your OS or follow the Wi
 
 Now you have +git+, +golang+, +rust+, and necessary libraries installed, let's get to work!
 
+[[Go-Ethereum]]
```

Comment:
Andreas point out in #279, _"I prefer to use descriptive names with **underscore** between words, up to 2-4 words in length"_.
IMHO, I'd also avoid letter capitalization.

Reply:
Good to know, I tried to see if there are any code conventions regarding this but couldn't find any. 
A question, I am trying to reference a paragraph in a different chapter (clients.ascii), this does not seem to work well with regular references. 
Meaning, if I reference something by using `<<refrence>>`, where `[[reference]]` is not in the same file, the reference does not work. 

The workaround I found is to use `<<clients.asciidoc#reference, ParagraphName>>` . Is there a better way to do this, or should I only reference the chapter?